swatch-core/schemas/instances_export_csv_item.yaml

```diff
@@ -36,3 +40,8 @@ properties:
     type: string
   hypervisor_uuid:
     type: string
+  number_of_guests:
+    type: integer
+  hardware_type:
+    description: (Only for guests) What the type of the host is (e.g. hypervisor, physical, etc.).
+    type: string
```

Comment:
Actually it'd be ideal to include hardware type for all hosts

Reply:
Nitpick: no newline at end of file

Reply:
PR updated with the newline at end of the file.